_includes/navbar.html

```diff
@@ -1,33 +1,37 @@
 <a id="skipnav" href="#main">Skip to main content</a>
 
-<header role="banner">
-  <div class="disclaimer">
-    <span class="us-official">
-      An official website of the United States Government.
-      <img
-      alt="US flag signifying that this is a United States Federal Government website"
-      src="{{ site.baseurl }}/assets/img/us_flag_small.png">
-    </span>
-    <span class="stage">This site is currently in <a href="https://18f.gsa.gov/dashboard/stages/#alpha">alpha</a>.</span>
-  </div>
-  <nav>
-    <a href="#" id="menu-btn">&#9776; MENU</a>
-    <div id="logo">
-      <a href="{{ site.baseurl }}/" accesskey="1" title="Home" aria-label="Home">
-        <h1>U.S. Federal Web Design Standards</h1>
-      </a>
+<header class="usa-site-header" role="banner">
+
+  <div class="usa-disclaimer">
+    <div class="usa-grid">
+      <span class="usa-disclaimer-official">An official website of the United States Government 
+        <img alt="US flag signifying that this is a United States Federal Government website" src="{{ site.baseurl }}/assets/img/us_flag_small.png">
+      </span>
+      <span class="usa-disclaimer-stage">This site is currently in alpha. <a href="https://18f.gsa.gov/dashboard/stages/#alpha">Learn more.</a></span>
     </div>
-    <ul class="usa-unstyled-list">
-      <li>
-        <a href="{{ site.baseurl }}/about/">About</a>
-      </li>
-      <li>
-        <a href="https://github.com/18F/usfwds/" target="_blank">
-          View on Github
+  </div>
+  
+    <nav class="usa-site-navbar">
+      <div class="usa-grid">
+      <a href="#" id="menu-btn">&#9776; MENU</a>
+      <div id="logo">
+        <a href="{{ site.baseurl }}/" accesskey="1" title="Home" aria-label="Home">
+          <h1 class="usa-header-logo-title">U.S. Federal Web Design Standards</h1>
```

Comment:
What's `.usa-header-logo-title` doing? I can't find it being used in the current codebase.